### DISCUSSION TITLE
Notifications: "you have 1 review request" shows nothing.

### DISCUSSION THREAD

**Discussion by Pomax** [community/community#63830] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

First off, it looks like the notifications page, which is an entire UI on its own, does not have a category, so that's probably worth creating.

That said, my notifications page says I have 1 pending review request, but shows nothing when actually clicked: 

![{E45A96DF-DCA9-4AC3-A31E-E266F61914AD}](https://github.com/community/community/assets/177243/0b2c7c4e-e92b-449e-b35d-f581f48e7e4a)

That's all I have, this is clearly a bug, that's all I can bring to the table here.

**Comment by jpcc0** [comment]
Hi,

Have you tried clearing your browser cache and cookies?

**Comment by Pomax** [comment]
Really not looking for community engagement here, just filing a bug in the place github now wants bugs filed.
